client/src/pages/RecipeDetailPage.jsx

```diff
@@ -1,7 +1,507 @@
+            }
+        ]
+    },
+}
 
 const RecipeDetailPage = () => {
-  return <div>This is the recipe detail page</div>;
+  const classes = useStyles();
+  return (
+    <>
+      <Header className="header">
+        <a href="/" className="header-app-name">App Name</a>
+        <a href="/recipe">Search by Recipe</a>
+        <a href="/ingredient">Search by Ingredient</a>
+      </Header>
+
+      <div className={classes.mainContainer}>
+        <Grid container spacing={3}>
+          <Grid item xs={1} />
+          <Grid item xs={10}>
+            <Paper className={classes.paper}>
+              <h1>{data.title}</h1>
+            </Paper>
+          </Grid>
+          <Grid item xs={1} />
+        </Grid>
+        <Grid container spacing={3}>
+          <Grid item xs={1} />
+          <Grid item xs={2}>
+            <Paper className={classes.paper}>
+              <h3>This dish goes well with:</h3>
+              {data.winePairing.pairedWines.map((wine, i) => (
+                <li key={i}>
+                  {wine}
+                </li>
+              ))}
+              {data.winePairing.productMatches.map((wine, i) => (
+                <a href={wine.link}><img src={wine.imageUrl} /></a>
+              ))}
+            </Paper>
+          </Grid>
+          <Grid item xs={4}>
+            <Paper className={classes.paper}>
+              <a href={data.sourceUrl}>
+                <img
+                  className={classes.recipeImage}
+                  src={data.image}
+                  alt={data.title}
+                />
+              </a>
+              <h3>
+
+                {' '}
+                Ready In:
+                {' '}
+                {data.readyInMinutes}
+                {' '}
+                Minutes
+              </h3>
+            </Paper>
+          </Grid>
+          <Grid item xs={4}>
+            <Paper className={classes.paper}>
+              <Typography variant="h5" gutterBottom>
+                <b>{`${data.aggregateLikes} Likes`}</b>
+                {' '}
+                <HeartIcon color="secondary" fontSize="medium" />
+              </Typography>
+              <Typography variant="h5" gutterBottom>
+                Tasty Score:
+                {' '}
+                <b>{data.spoonacularScore}</b>
+              </Typography>
+              <Typography variant="h5" gutterBottom>
+                Health Score:
+                {' '}
+                <b>{data.healthScore}</b>
+              </Typography>
+              <Typography variant="h5" gutterBottom>
+                Weight Watchers Smart Points:
+                {' '}
+                <b>{data.weightWatcherSmartPoints}</b>
+              </Typography>
+              <Typography variant="h5" gutterBottom>
+                Vegetarian:
+                {' '}
+                {data.vegetarian ? <SmileyIcon color="secondary" fontSize="medium" /> : <SadIcon color="primary" fontSize="medium" /> }
```

Comment:
Can we use checkmarks and X's instead? The frowning face confused me for a while at first glance. Same for the below two.